src/sentry/tagstore/base.py

```diff
@@ -56,7 +56,10 @@ class TagStorage(Service):
 
         'get_group_tag_keys_and_top_values',
 
-        'get_tag_value_qs',
+        'get_tag_value_paginator',
+        'get_group_tag_value_paginator',
+        'get_group_tag_value_iter',
```

Comment:
None of these can be really be used for comparison so we should make sure to disable comparisons on these methods in getsentry.